src/ZendeskApi.Client/Resources/DeletedTicketsResource.cs

```diff
@@ -0,0 +1,185 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+using ZendeskApi.Client.Converters;
+using ZendeskApi.Client.Exceptions;
+using ZendeskApi.Client.Extensions;
+using ZendeskApi.Client.Formatters;
+using ZendeskApi.Client.Models;
+using ZendeskApi.Client.Queries;
+using ZendeskApi.Client.Responses;
+
+namespace ZendeskApi.Client.Resources
+{
+    /// <summary>
+    /// <see cref="https://developer.zendesk.com/rest_api/docs/core/tickets"/>
+    /// </summary>
+    public class DeletedTicketsResource : IDeletedTicketsResource
+    {
+        private const string ResourceUri = "api/v2/deleted_tickets";
+        
+        private readonly IZendeskApiClient _apiClient;
+        private readonly ILogger _logger;
+
+        private readonly Func<ILogger, string, IDisposable> _loggerScope = LoggerMessage.DefineScope<string>(typeof(TicketsResource).Name + ": {0}");
+
+        public DeletedTicketsResource(IZendeskApiClient apiClient, ILogger logger)
+        {
+            _apiClient = apiClient;
+            _logger = logger;
+        }
+
+        public async Task<IPagination<Ticket>> ListAsync(PagerParameters pager = null)
```

Comment:
Is this the return type you meant to use?

Reply:
Ah; that should probably have been a `DeletedTicketsListResponse`; cheers!